crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -64,6 +64,24 @@ def services
     end
   end
 
+  api :POST, "/api/upgrade/controllers", "Orchestrate the upgrade of controller nodes"
```

Comment:
As alternative to this API, we could have "/api/upgrade/controller".

Such call would upgrade **next non-upgraded controller node**, and return some special value when there's no one left so the caller could use different API call to continue. This way, each call of  "/api/upgrade/controller" could be reasonably reflected in the UI. 

@MaximilianMeister @rsalevsky what do you think?